Automattic/wp-calypso
# Add Global sidebar footer

Related to 
- https://github.com/Automattic/dotcom-forge/issues/5628
- https://github.com/Automattic/wp-calypso/pull/87542
- https://github.com/Automattic/dotcom-forge/issues/5627

## Proposed Changes

This PR updates the global sidebar to include a footer that matches the latest design: pfsHM7-aY-p2. (part of https://github.com/Automattic/dotcom-forge/issues/5628)

The original implementation is ported from https://github.com/Automattic/wp-calypso/pull/86766. You can see the diff by un-selecting the first 2 commits: https://github.com/Automattic/wp-calypso/pull/87576/files/008f0b465481fb9ad3c7ba170a8738b876776dd5..1c5b8e9ae74b008d29629ddf31c42612bddbbaad.

<img width="318" alt="Screenshot 2024-02-19 at 13 11 24" src="https://github.com/Automattic/wp-calypso/assets/5287479/8a36ead9-fe22-4943-8153-5351b9b0ca70">

## Testing Instructions



- Open Calypso Live
- Go to /sites
- See how the sidebar looks like

## Pre-merge Checklist
